modyn/config/schema/pipeline.py

```diff
@@ -386,6 +386,17 @@ class TriggerConfig(BaseModel):
 # ---------------------------------------------------- EVALUATION ---------------------------------------------------- #
 
 
+EvalStrategyName = Literal["MatrixEvalStrategy", "OffsetEvalStrategy"]
+
+
+class EvalStrategy(BaseModel):
+    name: EvalStrategyName = Field(description="The name of the evaluation strategy that should be used.")
+    config: Dict[str, Any] = Field(
```

Comment:
@robinholzi just merged a PR taking care about this in many different places (avoid passing around raw dictionaries), so let's try to keep this approach if we can :D I think these will just be 2 tiny classes instead of dicts, so it should not be too much bloat